### ISSUE TITLE
Low password strength doesn't log error properly.

### ISSUE THREAD

**Issue by Jieiku** [secretkeysio/jelly-actix-web-starter#39]
register a user:

jake
jake@example.myexample
1z4v3j

the form does nothing... no error no nothing....

If you use a password like 1234 then it will atleast give you an error.

make a much longer password and the account is created, such as 3flyingmonkeys6zoos

I know this has to do with the password function because creation a longer password works.....

Also in the log it returns a code 400 until I make the password long enough:

![2022-01-13_20-39](https://user-images.githubusercontent.com/106644/149452592-b702bbdc-a6f6-4f3c-b5bb-680fc86b702d.png)

I was looking at the file `jelly/src/forms/password.rs` and I believe the issue is here:

```rs
        if estimate.score() <= 2 {
            if let Some(feedback) = estimate.feedback() {
                if let Some(warning) = feedback.warning() {
                    self.errors.push(format!("{}", warning));
                }

                self.hints = feedback
                    .suggestions()
                    .iter()
                    .map(|s| format!("{}", s))
                    .collect();
            }

            return false;
        }
```

What I think happens is if the score is low.... and it has an inkling as to why such as using a password 12345 or qwerty then it will be like dont use sequential keys blah blah blah.... but if your password is complex but simply too short such as `1z4v3j` then it probably still has a low score but no warning message to pass along....

I am going to try and add some debug to this logic that will print the score to the server output/log

**Comment by ryanmcgrath** [comment]
I appreciate the bug report, but please flesh this out more before putting it up here. Some form of logging or indication of what's stalling out is helpful.

**Comment by Jieiku** [comment]
sure I am editing it now, let me do a little more digging.

**Comment by Jieiku** [comment]
Confirmed:
![2022-01-13_20-50](https://user-images.githubusercontent.com/106644/149453436-4b09befe-a675-4f30-a23e-5237b81c85b6.png)

**Comment by ryanmcgrath** [comment]
Ah, so in this case does `feedback.warning()` return `None`? That would explain why no error is being triggered.

If so, just shove a generic warning in I guess.

**Comment by Jieiku** [comment]
ok, I will give that a shot and submit a pull request if it works, thank you.

**Comment by Jieiku** [comment]
What if we displayed the password score as a meter? it looks like zxcvbn returns a score of 1-5

This would allow people to know the strength of their password as they are typing it...
The only thing is I am not sure if this is possible without ajax, and I dont want any ajax/javascript

**Comment by Jieiku** [comment]
@ryanmcgrath I added some more debug:
![2022-01-13_21-05](https://user-images.githubusercontent.com/106644/149454770-74f79c72-1314-4dad-9c14-dd9f48ccafc4.png)

**Comment by Jieiku** [comment]
fixed via https://github.com/secretkeysio/jelly-actix-web-starter/pull/40

**Comment by ryanmcgrath** [comment]
> fixed via #40

Going to close this and we can track over there - appreciate it!
